kse/src/org/kse/gui/dialogs/resources.properties

```diff
@@ -496,6 +496,9 @@ DViewPrivateKey.jbFields.mnemonic    = F
 DViewPrivateKey.jbFields.text        = Fields
 DViewPrivateKey.jbFields.tooltip     = Display private key's fields
 DViewPrivateKey.jbOK.text            = OK
+DViewPrivateKey.jbExport.mnemonic    = X
+DViewPrivateKey.jbExport.text        = Export
+DViewPrivateKey.jbExport.tooltip     = Export private key
```

Comment:
Hi @jgrateron, and all,

Here is the French version:
```ini
DViewPrivateKey.jbExport.mnemonic    = X
DViewPrivateKey.jbExport.text        = Exporter
DViewPrivateKey.jbExport.tooltip     = Exporte la clef priv\u00E9e dans un fichier
```
to put on:
https://github.com/kaikramer/keystore-explorer/blob/cece66530ffec4888aee4b503d44af40a5cd6c6d/kse/src/org/kse/gui/dialogs/resources_fr.properties#L497-L498


- Would you a PR or it is enough?

Regards,
Th.

Reply:
Ready added
Greetings.